phys2bids/bids_units.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+LGR = logging.getLogger(__name__)
+
+unit_aliases = {
+                # ampere: electric current
+                'a': 'A', 'ampere': 'A', 'amp': 'A', 'amps': 'A',
+                # kelvin: thermodynamic temperature
+                'k': 'K', 'kelvin': 'K', 'kelvins': 'K',
+                # mole: amount of substance
+                'mol': 'mol', 'mole': 'mol',
+                # hertz: frequency
+                'hz': 'Hz', 'hertz': 'Hz',
+                '1/s': 'Hz', '1/second': 'Hz', '1/seconds': 'Hz',
+                '1/sec': 'Hz', '1/secs': 'Hz',
+                # newton: force, weight
+                'n': 'N', 'newton': 'N', 'newtons': 'N',
+                # pascal: pressure, stress
+                'pa': 'Pa', 'pascal': 'Pa', 'pascals': 'Pa',
+                # volt: voltage (electrical potential), emf
+                'v': 'V', 'volt': 'V', 'volts': 'V',
+                # degree Celsius: temperature relative to 273.15 K
+                '°c': '°C', 'celsius': '°C', '°celsius': '°C',
+                # second: time
+                'second': 's', 'seconds': 's', 'sec': 's',
+                'secs': 's',
+                '1/hz': 's', '1/hertz': 's', 's': 's'
+}
+
+# Init dictionary of aliases for multipliers. Entries are still lowercase
+prefix_aliases = {
+                    # Multiples - skip "mega" and only up to "tera"
+                    'da': 'da', 'deca': 'da', 'h': 'h', 'hecto': 'h',
+                    'k': 'k', 'kilo': 'k', 'g': 'G', 'giga': 'G', 't': 'T',
+                    'tera': 'T',
+                    # Submultipliers
+                    'd': 'd', 'deci': 'd', 'c': 'c', 'centi': 'c',
+                    'milli': 'm', 'm': 'm', 'µ': 'µ', 'micro': 'µ',
+                    'n': 'n', 'nano': 'n', 'p': 'p', 'pico': 'p',
+                    'f': 'f', 'femto': 'f', 'a': 'a', 'atto': 'a',
+                    'z': 'z', 'zepto': 'z', 'y': 'y', 'yocto': 'y',
+}
+
+
+def bidsify_units(orig_unit):
+    """
+    Read the input unit of measure and use the dictionary of aliases
+    to bidsify its value.
+    It is possible to make simple conversions
+
+    Parameters
+    ----------
+    unit: string
+        Unit of measure, might or might not be BIDS compliant.
+
+    Returns
+    -------
+    new_unit: str
+        BIDSified alias of input unit
+
+    Notes
+    -----
+    This function should implement a double check, one for unit and
+    the other for prefixes (e.g. "milli"). However, that is going to be tricky,
+    unless there is a weird way to multiply two dictionaries together.
+    """
+    # call prefix and unit dicts
+    new_unit = ""
```

Comment:
```suggestion
    new_unit = ''
```
I know it's _peccata minuta_, but since the rest of phy2bids (and the function) uses `'` for strings, it would be nice to maintain the same style.